docs/docs/guides/05-type-system/01-philosophy.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Philosophy
+tags:
+  - typescript
+  - type system
+  - type
+  - philosophy
+  - principles
+---
+
+### Context
+
+Since its release, Strapi's strength has laid in its **flexibility** and **extensibility**.
```

Comment:
```suggestion
Since its release, Strapi's strength has lain in its **flexibility** and **extensibility**.
```

Reply:
I hesitated between both, apparently both work, but I'll trust you on it